src/dotnet/commands/dotnet-publish/LocalizableStrings.resx

```diff
@@ -155,4 +155,7 @@ The default is to publish a framework-dependent application.</value>
   <data name="CmdNoLogo" xml:space="preserve">
     <value>Do not display the startup banner or the copyright message.</value>
   </data>
+  <data name="NoSelfContainedOptionDescription" xml:space="preserve">
+    <value>Publish your application without the .NET Core runtime.</value>
```

Comment:
```suggestion
    <value>Publish your application without the .NET Core runtime. A supported .NET Core runtime must be installed on the target system to run your application.</value>
```

@KathleenDollard for string review.

Reply:
I would like framework dependent here. How about

Publish your application as a framework dependent application without the .NET Core runtime. A supported .NET Core runtime must be installed to run your application.</value>

Reply:
I like yours better, thanks!